src/edu/csus/ecs/pc2/clics/API202306/CLICSProvider.java

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 1989-2024 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.clics.API202306;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.csus.ecs.pc2.VersionInfo;
+import edu.csus.ecs.pc2.services.core.JSONUtilities;
+
+/**
+ * CLICS Provider
+ * Contains information about the system providing the API feed (PC2, in this case)
+ * 
+ * @author John Buck
+ *
+ */
+
+public class CLICSProvider {
+
+    @JsonProperty
+    private String name;
+
+    @JsonProperty
+    private String version;
+
+//  @JsonProperty
+//  private CLICSProviderLogo [] logos;
+
+    /**
+     * Fill in API Provider information properties (for the version endpoint)
+     * 
+     * @param versionInfo
+     */
+    public CLICSProvider(VersionInfo versionInfo) {
+        name = "pc2";
```

Comment:
Should this perhaps be upper case ("PC2")? Seems a bit more formal...

Reply:
Sure.  Changed in next push.